frontend/public/locales/en/public.json

```diff
@@ -728,5 +728,11 @@
   "OK": "OK",
   "Last updated": "Last updated",
   "Disabled": "Disabled",
-  "Enabled": "Enabled"
-}
\ No newline at end of file
+  "Enabled": "Enabled",
+  "Log in to your account": "Log in to your account",
+  "Log in": "Log in",
+  "Login": "Login",
+  "Welcome to {{platformTitle}}": "Welcome to {{platformTitle}}",
+  "Log in with {{providerName}}": "Log in with {{providerName}}",
```

Comment:
This won't necessarily work for the OAuth server login page. See https://github.com/openshift/oauth-server/pull/71#discussion_r609880583

Reply:
> @sg00dwin Don't manually add these messages to the public.json file. IIRC `yarn i18n` will remove them when run. We need to add i18next-parser comments to one our of source files like

@spadgett is there a recommended file to add these parser comments? Or should I just pick one?